### DISCUSSION TITLE
Import script help

### DISCUSSION THREAD

**Discussion by dcpo** [ananthakumaran/paisa#192] [Q&A] [answered]
Hey,
I am trying to import my bank statement. The one thing I can't figure out is how to get the entry for the credit card payment.

Script i am using 
>'''
{{#if (isDate ROW.B "DD/MM/YYYY")}}
  {{date ROW.B "DD/MM/YYYY"}} {{ROW.B}}
  {{#if (or (eq (trim ROW.I) "0.00") (isBlank (trim ROW.I)))}}
    {{or (match ROW.D  Expenses:Food="ZOMATO" Expenses:Groceries="paytm"   Liabilities:CreditCard:UB="POS") 
     "Expenses:Unknown"}}      {{trim ROW.H}} INR
    Assets:Checking:Union
  {{else}}
    Assets:Checking:Union
    {{predictAccount prefix="Income"}}      -{{trim ROW.I}} INR
  {{/if}}
{{/if}}
'''

here is sample excel file
[unionbankofindiaV2.xlsx](https://github.com/ananthakumaran/paisa/files/14426870/unionbankofindiaV2.xlsx)

**Comment by ananthakumaran** [comment]
> The one thing I can't figure out is how to get the entry for the credit card payment.

could you elaborate your question. I tried your import script and it seems to work and you seem to have figured out how to do classify account etc

**Reply by dcpo** [reply]
hey,
trying to get entry  that show credit card expense adding it to Liabilities and amount due payment that happen that clear out Liabilities of that credit card 

maybe smoothing like this.

> 2022/04/04 04/04/2022
         Expenses:Unknown                       3000INR
         Liabilities:CreditCard:UB
    2022/05/03 03/05/2022
         Income:Refund:CreditCard:UB             -3000INR
         Liabilities:CreditCard:UB

right now i get this entry when i import with script. which does not clear the Amount Due credit card. 

> 2022/04/04 04/04/2022
    Liabilities:CreditCard:UB                   3000 INR
    Assets:Checking:Union
  2022/05/03 03/05/2022
    Expenses:Unknown                            3000 INR
    Assets:Checking:Union

**Reply by ananthakumaran** [reply, marked as answer]
```handlebars
{{#if (isDate ROW.B "DD/MM/YYYY")}}
{{date ROW.B "DD/MM/YYYY"}} {{ROW.D}}
  {{#if (regexpTest ROW.D "Credit card Payment")}}
        Liabilities:CreditCard:UB  {{amount ROW.H}} INR
        Assets:Checking:Union
  {{else if (or (eq (trim ROW.I) "0.00") (isBlank (trim ROW.I)))}}
  {{or (match ROW.D Expenses:Food="ZOMATO" Expenses:Groceries="paytm" Liabilities:CreditCard:UB="POS")
       "Expenses:Unknown"}} {{trim ROW.H}} INR
        Assets:Checking:Union
  {{else}}
        Assets:Checking:Union
       {{predictAccount prefix="Income"}} -{{trim ROW.I}} INR
  {{/if}}
{{/if}}
```

you can make use of `else if` to keep adding more variations.

**Reply by dcpo** [reply]
yeah import is working now with credit card entry
